src/infra/marketplace/marketplaceCmdRepo.go

```diff
@@ -619,13 +622,20 @@ func (repo *MarketplaceCmdRepo) uninstallUnusedServices(
 		serviceNamesToUninstallMap[serviceNameWithVersion.Name.String()] = nil
 	}
 
-	installedItems, err := repo.marketplaceQueryRepo.ReadInstalledItems()
+	readInstalledItemsDto := dto.ReadMarketplaceInstalledItemsRequest{
+		Pagination: dto.Pagination{
+			ItemsPerPage: 100,
```

Comment:
Why 100? Isn't this arbitrary? If you want all the items, use at least something safer, like 1000 or 10000.

Reply:
I decided to use 100 because I doubt we’ll face issues with this amount in the coming months, maybe even years. Still, I agree that working with a thousand is indeed safer.

And yes, you're correct about the reason, it’s to ensure I can collect all of them. I could have considered a slightly better solution, like using `-1` as a reserved number or even a reserved keyword like `ALL`, but I don't think now is the right time to worry about that.